answers/javaScript.md

```diff
@@ -1 +1,3 @@
 ## JavaScript
+* `null` vs `undefined` - Null means there is 'no value.' Undefined represents a variable that has not been declared or not given a value.
+	* What does `type of null` output? It returns 'object.'
```

Comment:
Nice good answer, all the way from florida too...  Not even part of the class, cheers